lib/semian.rb

```diff
@@ -316,9 +316,9 @@ def create_adaptive_circuit_breaker(name, **options)
     # Fixed parameters based on design document recommendations
     AdaptiveCircuitBreaker.new(
       name: name,
-      kp: 0.3, # Standard proportional gain
-      ki: 0.003, # Moderate integral gain
-      kd: 1.0, # Small derivative gain (as per design doc)
+      kp: 0.75, # Standard proportional gain
```

Comment:
@Aguasvivas22 I put the new K values. Are you ok with me merging them as part of this?